Workflow/Sources/StateMutationSink.swift

```diff
@@ -44,7 +44,7 @@ public struct StateMutationSink<WorkflowType: Workflow> {
     ///   - update: The `State` mutation to perform.
     public func send(_ update: @escaping (inout WorkflowType.State) -> Void) {
```

Comment:
thoughts on exposing the ApplyContext to StateMutationSink?

Reply:
i think that probably makes sense to do, but would prefer to tackle as a follow-up for scope-management reasons, unless you feel this will immediately be desired. i _think_ it's a straightforward plumbing change, and i may have had it implemented at one point but removed it.